Developer-Guide.md

```diff
@@ -353,6 +356,36 @@ $ popd
 $ rm -rf "${tmpdir}"
 ```
 
+# Install Hypervisor
+
+`qemu-lite` will be installed automatically if you're using the [OBS packaging](https://github.com/kata-containers/documentation/tree/master/install)
```

Comment:
Suggested rewriting: 

When setting up Kata using a [packaged installation method](https://github.com/kata-containers/documentation/tree/master/install#installing-on-a-linux-system), the `qemu-lite` hypervisor is installed automatically. For other installation methods, you will need to manually a suitable hypervisor.